### PR TITLE
Change master to main in the tutorial

### PR DESCRIPTION
**Pull request by jtoar**
This PR changes all references from "master" to "main". This is only for the tutorial, and only really for the "Deployment" section. @cannikin I don't think this'll break anything, but wanted your opinion to be sure.

### CODE REVIEW

**Comment by cannikin** [comment]
Yeah I didn't change these at the time because I wasn't sure how Netlify would behave...it defaults to `master` when you create a new site, but I don't know if that's because `master` is hardcoded into new Netlify site setup, or there only happened to be one branch and that's the one it used—it which case it will keep working with `main`.

Did you want to try a test and see if it works okay? If so then this is good to go!

**Comment by jtoar** [comment]
@cannikin Absolutely, I'll give it a test and report back.

**Comment by thedavidprice** [comment]
Wasn't the other concern here that git and github default to 'master'? E.g. a fresh installation + 'git init' yields a 'master' branch...

File: TUTORIAL.md [hunk 1]
```diff
@@ -2371,17 +2371,17 @@ With a little luck (and SCIENCE) it will complete successfully! You can click th
 
 Did it work? If you see "Empty" under the About and Contact links then it did! Yay! You're seeing "Empty" because you don't have any posts in your brand new production database so head to `/admin/posts` and create a couple, then go back to the homepage to see them.
 
-> If you view a deploy via the **Preview** button notice that the URL contains a hash of the latest commit. Netlify will create one of these for every push to master but will only ever show this exact commit, so if you deploy again and refresh you won't see any changes. The real URL for your site (the one you get from your site's homepage in Netlify) will show the latest deploy. See [branch deploys](#branch-deploys) below for more info.
+> If you view a deploy via the **Preview** button notice that the URL contains a hash of the latest commit. Netlify will create one of these for every push to main but will only ever show this exact commit, so if you deploy again and refresh you won't see any changes. The real URL for your site (the one you get from your site's homepage in Netlify) will show the latest deploy. See [branch deploys](#branch-deploys) below for more info.
```
Comment (cannikin) [inline TUTORIAL.md:2374]: I missed this in the original text, but can you turn this one into code-style, so it shows as `main` instead of just main.

**Comment by cannikin** [comment]
> Wasn't the other concern here that git and github default to 'master'? E.g. a fresh installation + 'git init' yields a 'master' branch...

Ahhh yes, that was the other one. Has anyone talked about git itself updating the default branch name? When you `git init` you get `master`. We're going to have to add a couple more steps to the instructions if you need to create a new branch `main` and then delete `master` before pushing to GitHub. And that's before we even know if Netlify will automatically work with `main`.

File: TUTORIAL.md (hunk 1 above)
Comment (jtoar) [reply TUTORIAL.md:2374]: Fixed in #234.

**Comment by jtoar** [comment]
If you do the name change before pushing up to GitHub, the Netlify part's easy (in "Branch to deploy", it'll just say main):

change-master-to-main-netlify

But as you guys mention, since git initializes you with master, the tutorial would have to tell users to run

```
git init
git add .
git commit -m 'First commit'
git branch -m main
git add remote origin ...
git push -u origin main
```

or something similar. Is that too much git for the tutorial, especially for new users?

**Comment by cannikin** [comment]
> But as you guys mention, since git initializes you with master, the tutorial would have to tell users to run...

So if we add those instructions, what happens when you setup your site on Netlify? Does it automatically pick `main` over `master`? Or does it make you pick a branch? If the later we should either:

* make sure to tell people to pick `main`, otherwise it will default to `master` and future commits won't deploy to the main URL
* include instructions for deleting the `master` branch so you only have `main`

**Comment by jtoar** [comment]
@cannikin Yes because `main` is the only branch that exists so Netlify just picks that one. The proposed instructions just renames `master` to `main` before pushing up to GitHub, and it gets pushed up as `main` too.

**Comment by cannikin** [comment]
Ahh that's right, you'd have `main` and `master` locally but GitHub would only have `main`. Cool!

**Comment by jtoar** [comment]
@cannikin I think you don't have `master` locally either unless I'm misunderstanding `git branch -m`. Here's me following all the instructions and the output:

```
~/projects/redwood/issues-and-prs/change-main-to-master$ git init
Initialized empty Git repository in /home/dominic/projects/redwood/issues-and-prs/change-main-to-master/.git/
~/projects/redwood/issues-and-prs/change-main-to-master$ git add .
~/projects/redwood/issues-and-prs/change-main-to-master$ git commit -m 'First commit'
[master (root-commit) d6f3b12] First commit
 37 files changed, 12129 insertions(+)
 create mode 100644 .editorconfig
 create mode 100644 .env.defaults
 create mode 100644 .gitignore
 create mode 100644 .nvmrc
 create mode 100644 .vscode/extensions.json
 create mode 100644 .vscode/settings.json
 create mode 100644 LICENSE
 create mode 100644 README.md
 create mode 100644 api/.babelrc.js
 create mode 100644 api/jsconfig.json
 create mode 100644 api/package.json
 create mode 100644 api/prisma/schema.prisma
 create mode 100644 api/prisma/seeds.js
 create mode 100644 api/src/functions/graphql.js
 create mode 100644 api/src/graphql/.keep
 create mode 100644 api/src/lib/db.js
 create mode 100644 api/src/services/.keep
 create mode 100644 babel.config.js
 create mode 100644 netlify.toml
 create mode 100644 package.json
 create mode 100644 prettier.config.js
 create mode 100644 redwood.toml
 create mode 100644 web/.babelrc.js
 create mode 100644 web/jsconfig.json
 create mode 100644 web/package.json
 create mode 100644 web/public/README.md
 create mode 100644 web/public/favicon.png
 create mode 100644 web/public/robots.txt
 create mode 100644 web/src/Routes.js
 create mode 100644 web/src/components/.keep
 create mode 100644 web/src/index.css
 create mode 100644 web/src/index.html
 create mode 100644 web/src/index.js
 create mode 100644 web/src/layouts/.keep
 create mode 100644 web/src/pages/FatalErrorPage/FatalErrorPage.js
 create mode 100644 web/src/pages/NotFoundPage/NotFoundPage.js
 create mode 100644 yarn.lock
~/projects/redwood/issues-and-prs/change-main-to-master$ git branch
* master
~/projects/redwood/issues-and-prs/change-main-to-master$ git branch -m main
~/projects/redwood/issues-and-prs/change-main-to-master$ git branch
* main
```

**Comment by cannikin** [comment]
Ahhh is `-m` a rename? So many git flags!

**Comment by jtoar** [comment]
I think so, but didn't actually check the [docs](https://git-scm.com/docs/git-branch#Documentation/git-branch.txt--m) till just now:
```
-m
--move
  Move/rename a branch and the corresponding reflog.
```

**Comment by cannikin** [comment]
Sorry that took so long to merge, it got lost in the shuffle!

**Pull request merged**